### DISCUSSION TITLE
About the parameter settings and principles of the Spurt.

### DISCUSSION THREAD

**Discussion by ichaochao9** [isce-framework/spurt#38] [Q&A]
Hi, developers,

I have been trying to process SAR data using the spurt unwrapping method. In the following figure, how should I understand the meanings corresponding to the default values of the parameters ‘t_cost_type’ and ‘t_cost_scale’？The default value for the scale factor is 100, does this apply to most data processing?
![WechatIMG8](https://github.com/user-attachments/assets/caafa95f-9a68-4835-8c0c-0ba1fc14be72)

Could you please provide some references to help me understand the principle of the spurt?

**Comment by piyushrpt** [comment]
We are still in the process of documenting the code. But you can see the usage in the docstrings of the different cost functions here:

https://github.com/isce-framework/spurt/blob/563ed29d80147852d3ed1563c7c8549c7b551a0c/src/spurt/mcf/utils.py#L189
https://github.com/isce-framework/spurt/blob/563ed29d80147852d3ed1563c7c8549c7b551a0c/src/spurt/mcf/utils.py#L233

It should have no impact on constant costs.

**Reply by ichaochao9** [reply]
Thank you for your response, I will try to learn and understand it.
